src/readqps.jl

```diff
@@ -652,6 +652,17 @@ function readqps(filename::String)
         fill!(uvar, Inf)
     end
 
+    # Print problem names
+    if verbose
+        println("Problem          : ", name)
+        println("Objective sense  : $objsense")
+        println("Objective name   : ", objname)
+        rhs_section_read && println("RHS              : ", rhsname)
+        ranges_section_read && println("RANGES           : ", rngname)
+        bounds_section_read && println("BOUNDS           : ", bndname)
+        println()
+    end
```

Comment:
What I meant by logging is to use `@info`, `@warn`, etc., without a `verbose` flag. That's more powerful because it can be redirected to a stream, a file, etc.